libkbfs/block_journal.go

```diff
@@ -574,6 +607,8 @@ func (j *blockJournal) removeReferences(
 		if count == 0 {
 			delete(j.refs, id)
 			if removeUnreferencedBlocks {
+				// TODO: Decrement j.blockBytes, but only if the block
```

Comment:
Yeah, I think it's fine to let the byte count deviate in this case. Maybe just comment the behavior.